web/src/components/schedule/subcomponents/thursday_schedule/class_grid/class_grid.js

```diff
@@ -0,0 +1,82 @@
+import { Box, Grid, Typography } from "@mui/material";
+import { useTheme } from "@emotion/react";
+import messages from "./messages.js";
+
+function ClassGrid() {
+  const theme = useTheme();
+  return (
+    <Box>
+      <Grid container spacing={4} columns={{ sm: 4, md: 12 }}>
+        {/* Headers */}
+        <Grid item xs={3} display={{ xs: "none", md: "block" }}>
+          <Typography variant="h4">{messages.classHeader}</Typography>
+        </Grid>
+        <Grid item xs={4} display={{ xs: "none", md: "block" }}>
+          <Typography variant="h4">{messages.descriptionHeader}</Typography>
+        </Grid>
+        <Grid item xs={4} display={{ xs: "none", md: "block" }}>
+          <Typography variant="h4">{messages.prerequisitesHeader}</Typography>
+        </Grid>
+
+        {/* WCS Foundations */}
+        <Grid item xs={3}>
```

Comment:
Dunno where my previous comment went but can we turn the details of each class into a re-usable component?

Reply:
extracted into reusable component